### PR TITLE
EDU-709 - misc page updates for GCP

### PR DESCRIPTION
**Pull request by MasonEgger**

### CODE REVIEW

File: docs/evaluate/temporal-cloud/pricing.mdx [hunk 1]
```diff
@@ -275,7 +275,7 @@ Basic support has a monthly fee starting from $200.
 
 **Can I purchase Temporal Cloud through my Amazon, Azure, or Google Cloud Platform marketplace?**
 
-You can purchase Temporal Cloud credits in the AWS Marketplace.
+You can purchase Temporal Cloud credits in the AWS Marketplace. GCP marketplace purchase coming soon.
```
Comment (jsundai) [inline docs/evaluate/temporal-cloud/pricing.mdx:278]: every sentence on one line 😊

Comment (fairlydurable) [reply docs/evaluate/temporal-cloud/pricing.mdx:278]: Talking about upcoming changes is inherently fragile, especially if a ticket hasn't been filed to address things when a feature is released. Checked with Brian and we generally do not include.

File: docs/troubleshooting/blob-size-limit-error.mdx [hunk 2]
```diff
@@ -42,6 +42,6 @@ There are multiple strategies you can use to avoid this error:
      1. Execute Activities in smaller batches within a single Workflow Task.
      2. Introduce brief pauses or sleeps (for example, 1ms) between batches.
 
-3. Consider offloading large payloads to an object store (for example, Amazon S3) to reduce the risk of reaching blob size limits:
+3. Consider offloading large payloads to an object store to reduce the risk of reaching blob size limits:
```
Comment (fairlydurable) [inline docs/troubleshooting/blob-size-limit-error.mdx:45]: Maybe (for example, a cloud-based store)? This edit seems to lose the sense of "cloud" trying to be non-provider specific.

File: docs/production-deployment/cloud/account-setup/namespaces.mdx [hunk 3]
```diff
@@ -317,9 +317,10 @@ Each Namespace in Temporal Cloud has two unique endpoints, both of which include
 To ensure the security of your data, all traffic to and from your Namespace is encrypted.
 However, for enhanced protection, you have additional options:
 
-- (recommended) Set up [PrivateLink](https://aws.amazon.com/privatelink/) by [creating a ticket for Temporal Support](/cloud/support#support-ticket).
-  If you don't host your Clients and Workers on AWS, you can create an AWS account to proxy traffic through.
-- Allow list, for outgoing network requests from your Clients and Workers, the [IP address ranges](https://docs.aws.amazon.com/vpc/latest/userguide/aws-ip-ranges.html) of the AWS region in which your Namespace is located.
+- (recommended) Set up private connectivity by [creating a ticket for Temporal Support](/cloud/support#support-ticket).
+- Allow list, for outgoing network requests from your Clients and Workers, the [IP address ranges](https://docs.aws.amazon.com/vpc/latest/userguide/aws-ip-ranges.html) of the cloud provider region in which your Namespace is located.
+  - [AWS IP address ranges](https://docs.aws.amazon.com/vpc/latest/userguide/aws-ip-ranges.html)
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/account-setup/namespaces.mdx:322]: You made me look.

When talking about cloud providers is the term normally uppercased in technical  communication?

In technical communication, terms like "Cloud Provider" or "Cloud Service Provider" are often uppercased to denote specific entities or categories, especially when referring to major players like Amazon Web Services (AWS), Microsoft Azure, or Google Cloud Platform (GCP). Using uppercase helps differentiate these terms from general usage. For instance:

Cloud Provider (when referring to a specific provider or category)
Cloud Service Provider (for a formal category)
However, in more general contexts, lowercase might be used if the term is not meant to specify a particular entity or if it’s used generically. For example:

"Our application can be deployed on any cloud provider."
Ultimately, the choice may depend on the style guide or conventions used by the organization or document.

File: docs/troubleshooting/blob-size-limit-error.mdx (hunk 2 above)
Comment (MasonEgger) [reply docs/troubleshooting/blob-size-limit-error.mdx:45]: I think Object Storage is universal enough that people know what it means, at least in cloud space. I think adding "cloud based store" doesn't add anything.

File: docs/evaluate/temporal-cloud/pricing.mdx (hunk 1 above)
Comment (MasonEgger) [reply docs/evaluate/temporal-cloud/pricing.mdx:278]: I'll remove it.

Comment (MasonEgger) [reply docs/evaluate/temporal-cloud/pricing.mdx:278]: Changing to `AWS Marketplace only.` as the question above does specifically call out GCP.

File: docs/production-deployment/cloud/account-setup/namespaces.mdx (hunk 3 above)
Comment (MasonEgger) [reply docs/production-deployment/cloud/account-setup/namespaces.mdx:322]: Upper cased.

**Review by XaF — APPROVED** [review]

**Comment by MasonEgger** [comment]
Ready for Replay release

File: docs/production-deployment/cloud/account-setup/namespaces.mdx [hunk 4]
```diff
@@ -317,9 +317,10 @@ Each Namespace in Temporal Cloud has two unique endpoints, both of which include
 To ensure the security of your data, all traffic to and from your Namespace is encrypted.
 However, for enhanced protection, you have additional options:
 
-- (recommended) Set up [PrivateLink](https://aws.amazon.com/privatelink/) by [creating a ticket for Temporal Support](/cloud/support#support-ticket).
-  If you don't host your Clients and Workers on AWS, you can create an AWS account to proxy traffic through.
-- Allow list, for outgoing network requests from your Clients and Workers, the [IP address ranges](https://docs.aws.amazon.com/vpc/latest/userguide/aws-ip-ranges.html) of the AWS region in which your Namespace is located.
+- (recommended) Set up private connectivity by [creating a ticket for Temporal Support](/cloud/support#support-ticket).
+- Allow list, for outgoing network requests from your Clients and Workers, the [IP address ranges](https://docs.aws.amazon.com/vpc/latest/userguide/aws-ip-ranges.html) of the Cloud Provider region in which your Namespace is located.
```
Comment (XaF) [inline docs/production-deployment/cloud/account-setup/namespaces.mdx:321]:
```suggestion
- Allow list, for outgoing network requests from your Clients and Workers, the IP address ranges of the Cloud Provider region in which your Namespace is located.
```

File: docs/troubleshooting/blob-size-limit-error.mdx (hunk 2 above)
Comment (fairlydurable) [inline docs/troubleshooting/blob-size-limit-error.mdx:45]: Exceeding?
```suggestion
3. Consider offloading large payloads to an object store to reduce the risk of exceeding blob size limits:
```

**Review by fairlydurable — APPROVED** [review]
Left some minor comments

**Review by jsundai — APPROVED** [review]

**Pull request merged**